### ISSUE TITLE
setting Notch value can result in one side with incorrect tab width 

### ISSUE THREAD

**Issue by michaelhauser** [kigster/laser-cutter#6]
Setting Notch value sometimes results in one side with incorrect tab width. An example is attached.

[makeabox-io-18.5x12.25x1.5-4183.box.pdf](https://github.com/kigster/laser-cutter/files/540800/makeabox-io-18.5x12.25x1.5-4183.box.pdf)

**Comment by kigster** [comment]
What do you mean incorrect? 

It's not meant to be exact, as the algorithm must break each side in equividistant sections. 

What did you expect to see, and what did you get exactly?

**Comment by michaelhauser** [comment]
The two long side panels are different and the one at the bottom of the PDF has slots that are too big for the tabs on the short side panels. Or am I missing something?

**Comment by kigster** [comment]
Yes, thank you — bug confirmed with another set of values: 5.5x2.0x6.0, notch 0.375, kerf 0.008.

**Comment by kigster** [comment]
![laser-cutter-issue-7-dimensions](https://cloud.githubusercontent.com/assets/259982/26526522/f8ab3f4a-4332-11e7-9e16-1bff6e831f04.png)
![makeabox-io-18_5x12_25x1_5-4183_box-details](https://cloud.githubusercontent.com/assets/259982/26526525/fe062e14-4332-11e7-8f92-ded14ce604e9.png)

**Comment by kigster** [comment]
Damn this bug :)

**Comment by hellocatfood** [comment]
Another set of values that causes an incomplete box
![screenshot from 2017-07-14 17-57-22](https://user-images.githubusercontent.com/151947/28222130-fb215c72-68bd-11e7-8aae-f32382c481ee.png)
[makeabox-90.00W-90.00H-90.00D-3.00T-20170714165535.pdf](https://github.com/kigster/laser-cutter/files/1149142/makeabox-90.00W-90.00H-90.00D-3.00T-20170714165535.pdf)

**Comment by kigster** [comment]
OK I think this is now fixed in version 1.0.5. Please test and report back. Thanks!
